priv/emqx.schema

```diff
@@ -593,6 +593,12 @@ end}.
   {datatype, {enum, [true, false]}}
 ]}.
 
+%% @doc Whether to ignore loop delivery of messages.(for mqtt v3.1.1)
```

Comment:
please update this comment accordingly as updated in `emqx.conf`

Reply:
I still think it's more clear to state `earlier than mqtt 5.0`. than being specific about `3.1.1`
also necessary to add a statement that `nl` from `mqtt 5.0` *IS* overridden by this config ?

Reply:
No, it is not necessary to state that overriden config because this change just have effect on mqtt3.0 message.